### PR TITLE
Update release process document

### PR DESCRIPTION
**Pull request by mr-v**
Some of the information in the release process wasn't up to date. Updates:

- Remove reference to ios-standup 562a0d3
- Update point of contact for dealing with complicated issues 42ceb0a
- Highlight that instructions are for manager 730ca99
- Update process for integrating hotfixes d783d4a
- Update info about reviewing changes after updating release branch with develop 83b42b1
- Reference develop branch in "Update the release calendar" tip. 4ad1e25

### CODE REVIEW

File: Cookbook/Technical-Documents/ReleaseProcess.md
Comment (viorel15) [inline Cookbook/Technical-Documents/ReleaseProcess.md]: Being the process we should just reference the role and not the name. If we want to keep the name then we can have a separate section where we can assign people to the roles (*release manager, platform QA, etc...)
```suggestion
1. These PRs need to be reviewed by the relevant squad or **platform QA** & one of the release engineers assigned to the release. This is to ensure visibility of changes being requested to be made, and to ensure the correct builds are available for validation.
```

**Review by NSMyself — APPROVED** [review]
LGTM 👍

File: Cookbook/Technical-Documents/ReleaseProcess.md
Comment (AnnKatF) [inline Cookbook/Technical-Documents/ReleaseProcess.md]:
```suggestion
	* In case there are changes other than updates to app and build versions, the release engineer should assign as reviewers all the engineers who worked on those changes and remove the ones automatically assigned by PullAssigners. There might be conflicts when merging the extra changes with `develop` or in the case of a hot fix PR not been merged previously to `develop`.
```

**Review by AnnKatF — COMMENTED** [review]
Looks good 👍 
Just added small suggestion.

File: Cookbook/Technical-Documents/ReleaseProcess.md
Comment (mr-v) [reply Cookbook/Technical-Documents/ReleaseProcess.md]: > There might be **conflicts** when merging the extra changes with `develop` or in the case of a hot fix PR not been merged previously to `develop`.

I was more trying to describe scenarios in which we have changes. It's very rare, since everything is supposed to be merged to `develop` soon after it gets to `release` branch. Maybe we can add even more details here to make it clear. See the following, what do you think?

- In case there are changes other than updates to app and build versions after merging the changes from `develop`, the release engineer should assign as reviewers all the engineers who worked on those changes and remove the ones automatically assigned by PullAssigners. In current workflow every change integrated to release branch is supposed to be go into to `develop` soon after, so differences should be minimal. Any differences might be a result of resolving conflicts or a hot fix PR not being merged previously to `develop`.

Comment (AnnKatF) [reply Cookbook/Technical-Documents/ReleaseProcess.md]: Sounds good 😃

File: Cookbook/Technical-Documents/ReleaseProcess.md
Comment (AliSoftware) [inline Cookbook/Technical-Documents/ReleaseProcess.md]: I'm thinking we should use a placeholder instead of hardcoded example value `4.15.0`, to avoid people copy/pasting that command and not realizing they need to update the branch name

```suggestion
      git log --format="%<(25)%an | %s" origin/develop..origin/release/{flavor}/{version} | grep -vE "^(Steve|iOS Bot) *\|" | sort
```

File: Cookbook/Technical-Documents/ReleaseProcess.md
Comment (mr-v) [inline Cookbook/Technical-Documents/ReleaseProcess.md]:
```suggestion
	*  In case there are changes other than updates to app and build versions after merging the changes from `develop`, the release engineer should assign as reviewers all the engineers who worked on those changes and remove the ones automatically assigned by PullAssigners. In current workflow every change integrated to release branch is supposed to be go into to `develop` soon after, so differences should be minimal. Any differences might be a result of resolving conflicts or a hot fix PR not being merged previously to `develop`.
```

**Review by viorel15 — APPROVED** [review]
👍

**Review by daniloaliberti — APPROVED** [review]
great one! ✨

**Review by AnnKatF — APPROVED** [review]
LGTM 🚀

**Pull request merged**
